src/sentry/models/rulesnooze.py

```diff
@@ -8,26 +9,43 @@
 @region_silo_only_model
 class RuleSnooze(Model):
     """
-    Duration an issue alert or metric alert is snoozed for a user. Null `until` value means snoozed forever.
+    Duration an issue alert or metric alert is snoozed for a user.
+    Null `until` value means snoozed forever.
+    Null `user_id` value means snoozed for all users.
     """
 
     __include_in_export__ = True
 
-    user_id = HybridCloudForeignKey("sentry.User", on_delete="CASCADE")
+    user_id = HybridCloudForeignKey("sentry.User", on_delete="CASCADE", null=True)
+    owner_id = HybridCloudForeignKey("sentry.User", on_delete="SET_NULL", null=True)
```

Comment:
It looks like the only `on_delete` options for a `HybridCloudForeignKey` are CASCADE and SET NULL so I put SET NULL since I don't want the row deleted if the user leaves, but I also actually wanted this to be not nullable which seems at odds with the `on_delete` option 🤔

Reply:
Yeah, I think it's fine to make this column nullable so that we can support `SET_NULL`